genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -100,6 +100,10 @@ rights for this to work.
 to work.
 |false
 
+|genie.jobs.users.activeJobsLimit
+|The maximum number of active jobs a user is allowed to have. Once a user hits this limit, jobs submitted are rejected. Setting this property to zero or a negative value means no limit is enforced.
+|0
```

Comment:
This is one way to do it another is a feature flag that enables or disables this. Do you have any reason you went with this over that? Setting to zero or negative is kind of counter intuitive as opposed to setting to true or false for enabled and then taking a setting...